library/CMakeLists.txt

```diff
@@ -66,6 +66,8 @@ set(src_crypto
     xtea.c
 )
 
+list(APPEND src_crypto ${thirdparty_src})
```

Comment:
Where is `thirdparty_src` defined? I'm not sure we need this line.

Also, if `src_crypto` is not used at all anymore, we should probably delete it from this file.

Reply:
It is defined and exported to parent scope in `crypto/3rdparty/CMakeLists.txt`. It is visible here because the main `CMakeLists.txt` pulls it in.

I noticed it and thought about removing `src_crypto` but considered it out of scope for this PR, like many similar repo-split activities that we haven't done yet.